pkg/ddevapp/ddevapp.go

```diff
@@ -939,6 +942,26 @@ func (app *DdevApp) GetAllURLs() []string {
 		}
 		URLs = append(URLs, "http://"+name+httpPort, "https://"+name+httpsPort)
 	}
+
+	dockerIP, err := dockerutil.GetDockerIP()
+	if err != nil {
+		util.Error("Unable to get Docker IP: %s", err)
```

Comment:
This would be better as a %v, but I'm concerned that we're not doing the right thing by continuing. Probably the signature of this function is wrong, should have an error. But it's probably OK anyway.